flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BucketWriter.java

```diff
@@ -95,7 +97,7 @@
   private final String onCloseCallbackPath;
   private final long callTimeout;
   private final ExecutorService callTimeoutPool;
-  private final int maxConsecUnderReplRotations = 30; // make this config'able?
+  private static final int maxConsecUnderReplRotations = 30; // make this config'able?
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00115)